DigitalOffice/Tests/ProjectServiceTests/Repositories/EditProjectByIdRepositoryTests.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Linq;
+using NUnit.Framework;
+using Microsoft.EntityFrameworkCore;
+using LT.DigitalOffice.Kernel.UnitTestLibrary;
+using LT.DigitalOffice.ProjectService.Database;
+using LT.DigitalOffice.ProjectService.Repositories;
+using LT.DigitalOffice.ProjectService.Database.Entities;
+
+namespace LT.DigitalOffice.ProjectServiceUnitTests.Repositories
+{
+    public class EditProjectByIdRepositoryTests
+    {
+        private Guid departmentId;
+
+        private DbProject dbProject;
+        private DbProject editProject;
+
+        private ProjectRepository repository;
+        private ProjectServiceDbContext dbContext;
+        private DbContextOptions<ProjectServiceDbContext> dbOptionsProjectService;
+
+        [SetUp]
+        public void SetUp()
+        {
+            dbOptionsProjectService = new DbContextOptionsBuilder<ProjectServiceDbContext>()
+                .UseInMemoryDatabase(databaseName: "ProjectServiceTest")
+                .Options;
+
+            dbContext = new ProjectServiceDbContext(dbOptionsProjectService);
+            repository = new ProjectRepository(dbContext);
+
+            departmentId = Guid.NewGuid();
+
+            dbProject = new DbProject
+            {
+                Name = "A test name",
+                DepartmentId = departmentId,
+                Description = "Description",
+                IsActive = true,
+                Deferred = false
+            };
+
+            editProject = new DbProject
+            {
+                Name = "Is different",
+                DepartmentId = Guid.NewGuid(),
+                Description = "Is different too",
+                IsActive = false,
+                Deferred = false
+            };
+        }
+
+        [Test]
+        public void SuccessfulEditProjectTest()
```

Comment:
My version of the test: 
![изображение](https://user-images.githubusercontent.com/61084995/90395108-9b1b5080-e09c-11ea-8c75-75073558734e.png)

Reply:
What do you imply here? I don't agree with this code.
There is no confirmation that the project was added initially.

And since this is the only visible difference between two versions of the test (Ah, also there's much less Assert-checks), I can't see how this is better or something.

Reply:
Subjectivity. Conversation closed.